src/driver/odoc_driver.ml

```diff
@@ -582,6 +582,20 @@ let run libs verbose packages_dir odoc_dir odocl_dir html_dir stats nb_workers
       (fun () -> render_stats env nb_workers)
   in
 
+  List.iter
```

Comment:
Is all of this just debug ? Otherwise, it seems too verbose and opaque.

Reply:
Yes, this is only showing filtered logs for debugging. I'll remove it.